iothub/device/tests/ClientSettings/TransportSettingsTests.cs

```diff
@@ -125,20 +126,20 @@ public void ConnectionPoolSettings_PoolingOff()
 
         [TestMethod]
         [ExpectedException(typeof(ArgumentException))]
-        public void IotHubDeviceClient_NullX509Certificate()
+        public async Task IotHubDeviceClient_NullX509Certificate()
```

Comment:
Possible noob question, but do async test methods need to have the word async in them? Or is that just for normal SDK class methods?

Reply:
If the method awaits other calls, it must have the keyword `async` in it, so yes.